### PR TITLE
perf: remove unnecessary null byte

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

followup to: #2101

### CODE REVIEW

**Review by gsk967 — APPROVED** [review]
LGTM

**Review by toteki — CHANGES_REQUESTED** [review]
discussion

File: x/leverage/keeper/iter.go [hunk 1]
```diff
@@ -177,12 +178,12 @@ func (k Keeper) SweepBadDebts(ctx sdk.Context) error {
...
 // GetAllUTokenSupply returns total supply of all uToken denoms.
 func (k Keeper) GetAllUTokenSupply(ctx sdk.Context) sdk.Coins {
-	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixUtokenSupply), store.NoLastByte)
+	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixUtokenSupply), keys.NoLastByte)
 }
 
 // GetAllReserves returns all reserves.
 func (k Keeper) GetAllReserves(ctx sdk.Context) sdk.Coins {
-	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixReserveAmount), store.NoLastByte)
+	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixReserveAmount), keys.NoLastByte)
```
Comment (toteki) [inline x/leverage/keeper/iter.go:186]: Doesn't this break mainnet? Leverage already stores these coins with null terminator

File: x/incentive/keeper/iter.go [hunk 2]
```diff
@@ -87,10 +87,10 @@ func (k Keeper) getAllAccountUnbondings(ctx sdk.Context) ([]incentive.AccountUnb
...
 // getAllTotalUnbonding gets total unbonding for all uTokens (used for a query)
 func (k Keeper) getAllTotalUnbonding(ctx sdk.Context) sdk.Coins {
-	return store.SumCoins(k.prefixStore(ctx, keyPrefixTotalUnbonding), store.NoLastByte)
+	return store.SumCoins(k.prefixStore(ctx, keyPrefixTotalUnbonding), keys.ToStr)
 }
 
 // getAllTotalBonded gets total bonded for all uTokens (used for a query)
 func (k Keeper) getAllTotalBonded(ctx sdk.Context) sdk.Coins {
-	return store.SumCoins(k.prefixStore(ctx, keyPrefixTotalBonded), store.NoLastByte)
+	return store.SumCoins(k.prefixStore(ctx, keyPrefixTotalBonded), keys.NoLastByte)
```
Comment (toteki) [inline x/incentive/keeper/iter.go:95]:
```suggestion
	return store.SumCoins(k.prefixStore(ctx, keyPrefixTotalBonded), keys.ToStr)
```

**Review by toteki — APPROVED** [review]

File: x/leverage/keeper/iter.go (hunk 1 above)
Comment (toteki) [reply x/leverage/keeper/iter.go:186]: ok

**Pull request merged**